tutorials/EleventyWebinyBlog/_site/about/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="/css/style.css" />
+  <title>About Me</title>
+</head>
+<body>
+    
+  <header><nav class="navbar is-light" role="navigation" aria-label="main navigation">
+  <div class="navbar-start">
+    <div class="navbar-item has-text-weight-bold">
+      My Eleventy Blog
+    </div>
+  </div>
+  <div class="navbar-end">
+    <a href="/" class="navbar-item">
+      Home
+    </a>
+
+    <a href="/about" class="navbar-item">
+      About Me
+    </a>
+  </div>
+</nav>
+</header>
+
+  <main>
+    
+<article class="page-layout">
+  <div class="container">
+    <div class="columns">
+      <div class="column is-8 is-offset-2">
+        <div class="content mt-5">
+          <h1>About Me</h1>
+          <p>I am a person that writes stuff.</p>
+
+        </div>
+      </div>
+    </div>
+  </div>
+</article>
+
+  </main>
```

Comment:
Please format the code, there are a lot of gaps and irregularities, here for example.